src/presentation/test/builder/GeneralPartnerBuilder.ts

```diff
@@ -86,22 +86,18 @@ class GeneralPartnerBuilder {
   }
 
   isPerson() {
-    generalPartnerPerson.forename = "Joe";
```

Comment:
this was updating the object outside of the builder so was not setting the fields on the data object being used by the builder.